index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <span class="my-logo"><a href="#">My Logo</a></span>
+        <nav>
+          <a href=""><img src="assets/images/Union.png" alt="Union-icon" /></a>
+        </nav>
+      </header>
+
+      <section class="hero">
+        <div class="greeting">
+          <p>
+            I'm Zerihun. <br />
+            Glad to see <br />
+            you!
+          </p>
+        </div>
+        <div class="job-description">
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <span class="connect-link">LET'S CONNECT</span>
```

Comment:
- [ ] Kindly consider adding this in a header tag to make this semantically correct and the screen readers and other technologies will thank you.